WordPress-Extra/ruleset.xml

```diff
@@ -2,6 +2,22 @@
 <ruleset name="WordPress Extra">
 	<description>Best practices beyond core WordPress Coding Standards</description>
 
+	<rule ref="Generic.PHP.DeprecatedFunctions"/>
+	<rule ref="Generic.PHP.ForbiddenFunctions"/>
+	<rule ref="Generic.Functions.CallTimePassByReference"/>
+	<rule ref="Generic.Formatting.DisallowMultipleStatements"/>
+	<rule ref="Generic.Formatting.MultipleStatementAlignment"/>
```

Comment:
> It is part of the standard.

Kinda. It says “spaces can be used mid-line for alignment”. It doesn't say that that alignment is required.